keyboards/jopr/keymaps/default/keymap.c

```diff
@@ -0,0 +1,59 @@
+        KC_LSFT,   KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,                  KC_P1,   KC_P2,   KC_P3,   KC_PENT,   KC_HOME, KC_UP,   KC_END, \
+        KC_LCTL,   KC_LWIN,   KC_LALT,   KC_SPC,                                                KC_RALT, KC_RWIN, LT(_FN,KC_MENU), KC_RCTL,     KC_P0,            KC_PDOT,            KC_LEFT, KC_DOWN, KC_RGHT \
+    ),
+    [_KANA] = LAYOUT(
+		_______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______,   _______, _______, _______, _______, \
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,           _______, _______, _______, _______, \
+		_______,     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, \
+		_______,       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,              _______, _______, _______,            _______, _______, _______, \
+		_______,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                  _______, _______, _______, _______,   _______, _______, _______, \
+		_______,   _______,   _______,   _______,                                                 _______,   _______,   _______,   _______,     _______,          _______,            _______, _______, _______ \
+    ),
+    [_FN] = LAYOUT(
+		_______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______,   KC_SYSREQ, _______, _______, _______, \
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,           _______, _______, _______, _______, \
+		_______,     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,       KC_MSTP, KC_VOLU, _______, _______, \
+		_______,       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,              KC_MPRV, KC_MPLY, KC_MNXT,            _______, _______, _______, \
+		_______,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                  KC_MUTE, KC_VOLD, _______, _______,   KC_MSTP, KC_VOLU, KC_MPLY, \
+		_______,   _______,   _______,   _______,                                                 _______,   _______,   _______,   _______,     _______,          _______,            KC_MPRV, KC_VOLD, KC_MNXT \
+    )
+};
+
+void led_set_user(uint8_t usb_led) {
+
+    /*if (usb_led & (1 << USB_LED_KANA)) {
```

Comment:
If this keyboard has no Kana LED capability, this block can be removed.

Reply:
It's a reminder for myself as to where the third LED is, which will be doing something similar to a Kana LED.